extensions/dark-mode-switcher/package.json

```
@@ -0,0 +1,56 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "dark-mode-switcher",
+  "title": "Dark Mode Switcher",
+  "description": "Toggles dark mode with a single command.",
+  "icon": "icon.png",
+  "license": "MIT",
+  "author": "taken",
```

Use your github handle
```suggestion
  "author": "TakenMC",
```